unit_tests/dynamic_object_mappings.py

```diff
@@ -5,13 +5,13 @@
 def test__dynamic_objects_mappings(fmc):
     logging.info("Testing Dynamic Objects Mappings")
 
-    obj1 = fmcapi.DynamicObjectMappings(fmc)
-    obj1.handle_mappings(action="add", value=["10.0.0.4"], name="Sales")
-    obj1.post()
+    # obj1 = fmcapi.DynamicObjectMappings(fmc)
```

Comment:
Is this intentionally commented out, or was commented out during testing? Either remove or reinstate